firewalls.tf

```
@@ -104,3 +108,28 @@ resource "digitalocean_firewall" "web" {
     source_addresses = ["0.0.0.0/0", "::/0"]
   }
 }
+
+
+resource "digitalocean_firewall" "census" {
+  name        = "web"
```

Failed to deploy with the error `Error: Error creating firewall: POST https://api.digitalocean.com/v2/firewalls: 409 (request "071e459a-4b88-4f2c-8429-fdbd9e9b1044") duplicate name`

Fix: 

```suggestion
  name        = "census"
```

Added in https://github.com/jenkins-infra/digitalocean/pull/266